### PR TITLE
Bring back py-wheel command for building test wheels

### PR DESCRIPTION
**Pull request by Wumpf**
### What

Got lost at some point but we still mention it in [our build instructions](https://github.com/rerun-io/rerun/tree/main/rerun_py#building-rerun-from-source).

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/6490?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/6490?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!

- [PR Build Summary](https://build.rerun.io/pr/6490)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

File: pixi.toml [hunk 1]
```diff
@@ -214,6 +214,18 @@ py-docs-serve = "mkdocs serve -f rerun_py/mkdocs.yml -w rerun_py"
 # Lint markdown
 mdlint = "python scripts/ci/mdlint.py"
 
+[target.win-64.tasks]
+# Builds a debug mode Python wheel for install in an environment.
+py-wheel = { cmd = "cp target/debug/rerun.exe rerun_py/rerun_sdk/rerun_cli/rerun.exe && pixi run maturin build --manifest-path rerun_py/Cargo.toml --no-default-features --features pypi", depends_on = [
+  "py-build",
```
Comment (jleibs) [inline pixi.toml:220]:
```suggestion
  "rerun-build-web-cli",
```
This doesn't actually depend on py-build. Also pypi activates web feature, so we need to build the web stuff.

File: pixi.toml [hunk 2]
```diff
@@ -214,6 +214,18 @@ py-docs-serve = "mkdocs serve -f rerun_py/mkdocs.yml -w rerun_py"
...
+] }
+
+[target.unix.tasks]
+# Builds a debug mode Python wheel for install in an environment.
+py-wheel = { cmd = "cp target/debug/rerun rerun_py/rerun_sdk/rerun_cli/rerun && pixi run maturin build --manifest-path rerun_py/Cargo.toml --no-default-features --features pypi", depends_on = [
+  "py-build",
```
Comment (jleibs) [inline pixi.toml:226]:
```suggestion
  "rerun-build-web-cli",
```

**Review by jleibs — APPROVED** [review]
Makes sense, but the dep needs to be fixed.

File: pixi.toml (hunk 1 above)
Comment (Wumpf) [reply pixi.toml:220]: We rather need both, right?

Comment (jleibs) [reply pixi.toml:221]: Not as far as I'm aware. 

`maturin dev` and `maturin build` are independent commands. They both cause the bindings to get build but then package the artifacts in different ways. 

`py-build` should really be called `py-dev`

**Comment by emilk** [comment]
bump

**Comment by roym899** [comment]
I just tried this in a clean environment. When using only `rerun-build-web-cli` the native viewer is not included.

**Comment by Wumpf** [comment]
needs another fresh attempt, pixi.toml changed a bit since then

**Pull request closed without merging**